### DISCUSSION TITLE
issue with committing/push from XCode

### DISCUSSION THREAD

**Discussion by ckunchur** [CS342/.github#29] [Technical Questions] [answered]
### In what area do you have a technical challenge?

Xcode, Simulator & Previews

### Description

I am able to successfully commit and push changes on my branch from command line, but when I try to do so in XCode I get the following error message: 
<img width="288" alt="image" src="https://user-images.githubusercontent.com/67437371/217711904-cbe39f55-0c78-4791-b3b1-679bf06564cc.png">

Was curious to know what the source of this could be?

### Reproduction

N/A

### Expected behavior

N/A

### Additional context

N/A

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment, marked as answer]
Hi @ckunchur,
The problem is coming up as we have used git ifs for the Template Application. We will revert that change so it will no longer be required to use git ifs for using the template app as well as your project. The following PR will already fix that for you: https://github.com/CS342/2023-PAWS/pull/4
Once this is merged, you should be able to clone the project from Xcode without the need to install git lfs.

In general, git lfs is a very nice solution to speed up the cloning process in git repos. You might find it useful in a later stage of your project. You can read more about it here: https://git-lfs.com
